src/ctia/flows/crud.clj

```diff
@@ -198,15 +210,18 @@
              store-fn
              identity
              entities
-             long-id-fn]}]
+             long-id-fn
+             spec]}]
```

Comment:
Good question, and I'm not sure.  Here is my thinking.  We do a round of validation with the schemas, and then maybe do another round of validation with the specs.  Since there is a performance hit, we can chose to only do the spec validation when there is a good reason.  For now, the only extra benefit is in the Reference spec, which only really applies to the Relationship entity.